tekton/publish.yaml

```diff
@@ -36,8 +36,8 @@ spec:
     command: ["mkdir"]
     args:
     - "-p"
-    - "/workspace/output/bucket/dashboard/latest/"
-    - "/workspace/output/bucket/dashboard/previous/"
+    - "/workspace/output/bucket/latest/"
```

Comment:
Hmmmm how come we don't want the subfolder here any more? Is this to do with the resolve path business we discussed?

Reply:
because the /dashboard is specified on the gs:// bucket, I removed it here. Keeping both leads to paths like /dashboard/dashboard/latest/release.yaml etc.